app/src/test/java/com/onboarding/parkingsystemkotlin/mvp/presenter/ConfigureParkingPresenterTest.kt

```diff
@@ -0,0 +1,45 @@
+package com.onboarding.parkingsystemkotlin.mvp.presenter
+
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import com.nhaarman.mockito_kotlin.whenever
+import com.onboarding.parkingsystemkotlin.listener.OnInputListener
+import com.onboarding.parkingsystemkotlin.mvp.contract.ConfigureParkingLotsContract
+import org.junit.Before
+import org.junit.Test
+
+private const val PARKING_LOT_STRING: String = "4"
```

Comment:
use a companion object for these constants pls

Reply:
and put the companion at the end of the file pls :D

Reply:
done